packages/admin-next/dashboard/src/v2-routes/regions/region-create/components/create-region-form/create-region-form.tsx

```diff
@@ -78,7 +78,7 @@ export const CreateRegionForm = ({
 
   const { t } = useTranslation()
 
-  const { mutateAsync, isLoading } = useCreateRegion()
+  const { mutateAsync, isPending } = useCreateRegion()
```

Comment:
I guess we need to switch to isPending everywhere?

Reply:
Yes, isLoading was renamed to isPending in v5.